# Author: Damyan Ivanov <dmn@debian.org>
# Description: fix methid declaration
#  GCC 4.1 brafs if it is fully qualified
--- a/src/trayicon.h
+++ b/src/trayicon.h
@@ -38,7 +38,7 @@ public:
 	TrayIcon(DictGui *win);
 	~TrayIcon();
 	
-	Gtk::Window *TrayIcon::getWindow();
+	Gtk::Window *getWindow();
 	
 private:
 	virtual bool on_button_press(GdkEventButton* event);
